.gitignore

```diff
@@ -33,3 +33,5 @@ logs/
 build/
 out/
 !gradle/wrapper/*.jar
+
+gradle.properties
```

Comment:
Can you describe, why was this added to the gitignore?

Reply:
It's a common Gradle file that, by convention, should be available in VCS. Please remove it from .gitignore. To resolve your problem, you can set up gradle.properties locally. You can place it in the Gradle user home, which is defined by the GRADLE_USER_HOME environment variable. If it's not set, it defaults to USER_HOME/.gradle. Create a file named gradle.properties there and fill it with the data you want.

Reply:
Resolved in [6909d92](https://github.com/SpaiR/imgui-java/pull/211/commits/6909d92c1d9f85fee43a73e5f6a587d3fb9896b1)